src/nflxmultisubs.js

```diff
@@ -14,14 +14,45 @@ const hookJsonParseAndAddCallback = function(_window) {
       if (result && result.result && result.result.movieId) {
           const movieId = result.result.movieId
           //console.log(`Intercepted manifest ${movieId}`);
-          window.__NflxMultiSubs.updateManifest(result.result);
+          // clone manifest so we can modify it w/o breaking player
```

Comment:
If we don't clone manifest here, we might get a chance to use the manifest maintained by player (same object), which contains all download urls of subtitles that have been used as primary subtitle.

But sometimes it just doesn't work, so I still implemented the merge logic.